workflow/executor/pns/pns.go

```diff
@@ -157,32 +157,6 @@ func (p *PNSExecutor) Wait(ctx context.Context, containerNames []string) error {
 		time.Sleep(1 * time.Second)
 	}
 
-OUTER:
```

Comment:
I don't think it is worthwhile getting this to work as the performance benefit is much smaller than the bugginess.